docs/developer-guide/working-with-rules.md

```diff
@@ -720,6 +720,8 @@ Rule   | Time (ms) | Relative
 quotes |    18.066 |   100.0%
 ```
 
+The environment variable can also be set to other values such as `TIMING=50` or `TIMING=all` to show a longer list of results.
```

Comment:
Can we provide a bit more detailed specification for this feature?

Reply:
I reworded the sentence for a bit more clarity. Let me know if there's some specific important detail missing. It seems concise and straightforward to me.